bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -268,6 +268,8 @@ BCStateTran::BCStateTran(const Config &config, IAppState *const stateApi, DataSt
   // callbacks.
   addOnTransferringCompleteCallback(
       [this](uint64_t checkpoint_num) { replicaForStateTransfer_->onTransferringComplete(checkpoint_num); });
+
+  LOG_INFO(getLogger(), "ST throughput measurement " << KVLOG(config_.getMissingBlocksSummaryWindowSize));
```

Comment:
This is not needed, we already print the whole config by implementing config operator<<().
I think it should be removed.